src/main/java/com/jfrog/ide/idea/scan/ScanBinaryExecutor.java

```diff
@@ -32,16 +36,37 @@ public abstract class ScanBinaryExecutor {
     protected List<String> supportedLanguages;
     private final boolean shouldExecute;
 
+    private static final String ENV_PLATFORM = "JF_PLATFORM_URL";
+    private static final String ENV_USER = "JF_USER";
+    private static final String ENV_PASSWORD = "JF_PASS";
+    private static final String ENV_ACCESS_TOKEN = "JF_TOKEN";
+
+
     ScanBinaryExecutor(String scanType, String binaryName) {
         this.scanType = scanType;
         if (SystemUtils.IS_OS_WINDOWS) {
             binaryName += ".exe";
         }
         Path binaryPath = BINARIES_DIR.resolve(binaryName);
-        commandExecutor = new CommandExecutor(binaryPath.toString(), Maps.newHashMap());
+        commandExecutor = new CommandExecutor(binaryPath.toString(), creatEnvWithCredentials());
         shouldExecute = Files.exists(binaryPath);
     }
 
+    private Map<String, String> creatEnvWithCredentials() {
+        Map<String, String> env = new HashMap<>();
```

Comment:
Do you know if the binaries require some environment variables such as `USER_HOME`, `PATH`, etc?
If not, please ignore this comment.
```suggestion
        Map<String, String> env = new HashMap<>(EnvironmentUtil.getEnvironmentMap());
```